packages/application-shell/src/components/project-switcher/project-switcher.spec.tsx

```diff
@@ -13,16 +13,10 @@ const render = () => {
       getIsExpired: (key) => key === 'key-3',
     }),
   ];
-  return renderApp(
-    <>
-      <label id="project-switcher">{'Project switcher'}</label>
-      <ProjectSwitcher projectKey="key-0" />
-    </>,
-    {
-      disableRoutePermissionCheck: true,
-      mocks: mockedRequest,
-    }
-  );
+  return renderApp(<ProjectSwitcher projectKey="key-0" />, {
+    disableRoutePermissionCheck: true,
+    mocks: mockedRequest,
+  });
```

Comment:
We don't need to render a label anymore since now the component itself renders one.